tests/install_upgrade_operators/strict_reconciliation/test_hco_related_objects.py

```diff
@@ -11,9 +11,13 @@
 class TestRelatedObjects:
     @pytest.mark.polarion("CNV-9843")
     def test_no_new_hco_related_objects(self, hco_status_related_objects):
-        assert len(hco_status_related_objects) == len(ALL_HCO_RELATED_OBJECTS), (
-            f"Expected related objects: {ALL_HCO_RELATED_OBJECTS}, actual: {hco_status_related_objects}"
-        )
+        actual_related_objects = {
+            (related_object["name"], related_object["kind"]) for related_object in hco_status_related_objects
+        }
+        expected_related_objects = {tuple(related_object.items()) for related_object in ALL_HCO_RELATED_OBJECTS}
```

Comment:
This is creating some weird dictionary, are you sure you want it this way? 
`{(('hyperconverged-cluster-cli-download', 'Route'),), (('kubevirt-console-plugin', 'Deployment'),), (('virtio-win', 'ConfigMap'),)....}`